oaknational/Oak-Web-Application
# Feat/lesq 722/specialist canonical pages [LESQ-722]

## Description

Music year: 2017

- add a canonical specialist lesson query
- update canonical lesson page to use this query
- remove 'lesson appears in' from canonical page due to complexity of types - will add another ticket to add this back in

## How to test

1. Go to https://deploy-preview-2339--oak-web-application.netlify.thenational.academy/teachers/lessons/sensory-story-a-christmas-carol-cgvk0t
3. You should see the canonical lesson page for specialist
